### ISSUE TITLE
Open Digital Preservation with Permanent.org

### ISSUE THREAD

**Issue by omnignorant** [open-austin/project-ideas#132]
### What problem are you trying to solve?

Public records are at risk – security breaches, natural disasters, technological shifts and a lack of legacy planning threaten the perpetual accessibility and integrity of digital data.

The vast majority of civic agencies, small government units, nonprofits, schools and other public sector organizations do not have a trusted place to store their digital materials. They rely on cumbersome, low-budget solutions from internal IT departments or pay for-profit internet giants recurring fees.

The digital storage industry was designed for convenience and short cycles. A commitment to permanent storage has no place in their business model.

### Who will benefit (directly and indirectly) from your project?

All citizens will benefit when the civil society organizations they trust have their own trusted agency for the security and preservation of critical public data and information.

### What other resources/tools are currently serving the same need? How does your project set itself apart?

Dropbox, Google Drive, Box, iCloud, Facebook – these are the cloud solutions the vast majority of Americans rely on to secure their digital materials, including nonprofits, schools and small government agencies. Less consumer friendly products for commercial license include Preservica. More savvy IT departments may host their own cloud solutions on internal servers. 

There are other open source projects out there that do this work. None aim to make content preservation and distribution accessible to the average, nontechnical user.

There are a ton of similarly spirited projects, these include: Filecoin, Interplanetary File System, Archmission, MOM, The Long Now Foundation, The Internet Archive, One Second Everyday, etc. that each take on a facet of the problem or propose even more visionary solutions to permanent storage. Again, none make an effort to address the individual, nontechnical, private user case.

### Where can we find any research/data available/articles?

David Rosenthal is pretty obsessed with this stuff: https://blog.dshr.org/

### What help do you need now?

We're building out our open API and our framework for viewing materials. It's a lot like the wordpress ecosystem, so we're open to exploring how to utilize wordpress front end, widgets or add ons.

### What are the next steps (validation, research, coding, design)?

Answer here.

### How can we contact you outside of Github(list social media or places you're present)?

Answer here.

----

#### Checklist for NEW ideas :baby:
Hey, you're official! You're now part of the growing civic hacking community in Austin. Here's a few things to get started (a couple you've probably already done).

- [x] Create this idea issue
- [x] Flesh out the who, where, and what questions above
- [x] Start the conversation about this idea on [Slack](https://openaustin-slackin.herokuapp.com/) Replace this link to the [#oa-permanent](https://open-austin.slack.com/messages/CCQVDH7UY) channel with your project's preferred channel.

#### Checklist for ACTIVE projects :fire:
Let's get this project started! When this idea starts taking off, the Projects Core Team will start helping this project's lead(s) out with project management and connecting you to resources you may need. To get there, please complete and check off the following:

- [x] Post an update at least once a month to this issue. Use [BASEDEF](https://github.com/open-austin/cheatsheet-basedef) for ideas, but it's ok even if your update is just "nothing new happened this month" or "we saw a small increase in traffic to our app this month". If there's no activity for two months, that's no problem, life happens. We'll just label this as `backlog` so others know you'll get back to it when you have the time. If nobody hears from you at all in more than two months, we may mark it as `abandoned` so that others can pick up this idea and run with it.
- [x] Take 30 minutes to complete [Open Leadership 101](https://mozilla.teachable.com/p/open-leadership-101) 
- [x] Create a [GitHub repository](https://github.com/Permanent-Legacy-Foundation). Ask for help setting up permissions if you want your repository to be within the [Open Austin Github organization](https://github.com/open-austin).
- [x] Create a [README file](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/) in your project repository. This file should help newcomers understand what your project is, why it's important, and kinds of help you're looking for.
- [x] [Create issues](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/project-set-up-for-collaboration-with-github/#assignment--add-your-first-issue) to describe each task that you plan to do or need help with and how a contributor can get started on that task. You might start and stop a lot, so consider issues as your to-do list.
- [x] Create a [team](https://help.github.com/articles/creating-a-team/) for your core contributors
  - This will make it easier for you to manage your github repo access. People on a team have the same level of access. Admin access will allow your trusted contributors to make changes as needed. 
  - You can remove and add people to your team as needed. 
  - View some of our teams: [https://github.com/orgs/open-austin/teams](https://github.com/orgs/open-austin/teams)
  - Note: You can also allow collaborators outside of your team and give them more limited access.
- [ ]   Create a [user group](https://get.slack.help/hc/en-us/articles/212906697-User-Groups) in Slack so you can "@" your core contributors all at once, without bothering other people who use the Slack channel. You'll need permission from a Slack admin, so just mention @leadership on Slack to get this set up.
- [ ]   Create a [Google Drive](https://drive.google.com), [Dropbox](https://www.dropbox.com/), or other cloud storage to share larger files. Github and Data.World are good for code and data, respectively, especially when you need version control. But they're not good for very large files, documentation, articles, etc. A cloud storage option will allow you to easily share, create, and collaborate on documents with your team and help organize ideas and thoughts.
  - Doing this early on can help your team stay organized and to onboard new contributors who wouldn't have access to files you all have shared over email.

#### Checklist for FEATURED Projects :tada:
To have your project FEATURED on [Open-Austin.org](https://open-austin.org/), complete the following documentation. In past projects, well-documented featured projects have more contributions than other projects.

- [ ] In your README, link to the [Open Austin Code of Conduct](https://www.open-austin.org/about/#code-of-conduct) or write your own code of conduct.
- [ ] [Create file: LICENSE](http://choosealicense.com/) to give your project an open license, allowing for sharing and remixing.
- [ ] [Create file: CONTRIBUTING.md](https://github.com/acabunoc/mozsprint-repo-template/blob/master/CONTRIBUTING.md) so others know [how they can contribute](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-contributor-guidelines/).
- [ ] Create an easily shareable project management artifact, like a [Civic Tech Canvas](https://cityofaustin.github.io/civic-tech-canvas/) or [Open Canvas](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas/)
- [ ] [Create an issue](https://github.com/open-austin/open-austin.github.io/issues/new) on the open-austin.github.io repo with the title `Add [my project] to projects page`. An Open Austin leader will review this issue and post your project :balloon:
- [ ] [Tell the City of Austin](http://austintexas.gov/page/mobile-app-library). If your idea is in a shareable format and can benefit people around the city, go to that site and follow the instructions on the bottom of the page to showcase your work there.

If you get stuck at any point, feel free to reach out to the leadership team on Slack by adding @leadership to your message. We're here to help you make real changes to our city.
